### DISCUSSION TITLE
Inconsistency with lancer-data README documentation and implementation in lancer-data?

### DISCUSSION THREAD

**Discussion by sealtrowitz** [massif-press/compcon#2298] [Archive]
I apologize if this is the wrong spot for this, but I wasn't sure where else to ask -- I've been helping my friend build an LCP for his homebrew and noticed a bit of an inconsistency with the lancer-data README documentation and actual implementation of said feature in lancer-data itself. Specifically, under the [Deployables](https://github.com/massif-press/lancer-data#deployables) header, it mentions that:

"A `recall` value will generate a recall action, removing the deployable from the user's Deployables list and refunding the system the `cost` value in charges. `deactivation` does the same as `recall` but does not refund the charge cost."

...But then checking the Hydra's Tempest Drone and Assassin Drones, both of which are non-Limited deployable systems, they use `recall` as opposed to `deactivation` in lancer-data. I went through and checked the rest of the non-Limited deployables in lancer-data I could think of (Lotus Projector, Hive Drone, Sentinel Drone, Tempest Drone, Assassin Drone, and Pattern-A Jericho Deployable Cover is the complete list I looked at; I did not look at Perimeter Command Plate since it explicitly cannot be recalled except through being destroyed or deploying a new one) and they all also use `recall`.

Is this something I should report on the lancer-data repo, either in regards to the README documentation or those items? Or am I just misunderstanding what `cost` is (I thought it specifically was an item's Limited cost)? Or am I maybe being unnecessarily pedantic with something that doesn't appear to actually be causing problems?

**Comment by jarena3** [comment]
I'm not sure I 100% follow -- these systems aren't correctly generating deactivation actions, they're instead generating erroneous recall actions? Is that correct?

**Reply by sealtrowitz** [reply]
Sorry, I'm probably not explaining it very well. I guess the core of it is, what's the difference between `deactivation` and `recall` for deployables? My understanding of the documentation is that `recall` refunds the `cost` value, but `deactivation` does not, so I thought that would mean `deactivation` should be used for non-limited deployable systems, but that isn't the case in the core data.

**Reply by jarena3** [reply]
ah, then yes your understanding is correct and that is a bug -- `recall` should refund limited usage and `deactivation` should not. Would you mind creating a ticket for this in lancer-data. It may also require changes in compcon (but I will address that seperately)

thanks for running this down!

**Reply by sealtrowitz** [reply]
I can do that, yeah! I've been busy the past week so I haven't had the chance to yet, but I will do that when I'm able!

**Reply by sealtrowitz** [reply]
Alright, finally was able to get that submitted at https://github.com/massif-press/lancer-data/issues/192 , sorry for the delay!

**Reply by jarena3** [reply]
don't worry about it at all! thank you!
